### ISSUE TITLE
Copy for enable tracking confusing

### ISSUE THREAD

**Issue by seb86** [woocommerce/woocommerce#23208]
![tracking](https://woocommerce.com/wp-content/uploads/2019/03/WooCommerce-3.6-Usage-Tracking1.png)

The copy for this option does not read well. I'm dyslexic and this confuses me. I had to read it 3 times slowly just to understand it.

Please consider changing it to the following please. 

> Do not check this box if you do not wish to opt-in for WooCommerce.com to track your usage data. We will not know you store exists or provide you recommendations.

**Comment by DavidAnderson684** [comment]
> Do not check this box if you do not wish to opt-in for WooCommerce.com to track your usage data. We will not know you store exists or provide you recommendations.

That's still a double-negative. Positive phrasing always requires less cognitive effort, e.g.

> Check this box if you wish to opt-in for WooCommerce.com to track your usage data.

Given that by default WP sends your URL and number of users (which in a lot of cases will be a 99%-good approximation to customers) to servers that Automattic have access to every 24 hours, is the other bit all strictly true?

> We will not know you store exists or provide you recommendations.

**Comment by seb86** [comment]
So really this option to enable is already enabled by WordPress and there is no way to fully opt-out. That is stupid and wrong.

**Comment by DavidAnderson684** [comment]
8-year-old Trac ticket: https://core.trac.wordpress.org/ticket/16778

Plugin - https://wordpress.org/plugins/update-privacy/ - though, it fails to exclude the site URL being sent in the `User-Agent` HTTP header.

**Comment by seb86** [comment]
Wow. Just... wow They prepare everything for the GDPR but to allow the site owner to opt-out or at least choose what data not to send back. Thanks for the plugin.

**Comment by timmyc** [comment]
@seb86 thanks for opening this issue, and sorry the wording is confusing. I'm going to take your suggestion and ask for assistance from our editorial team to make the wording more precise.

As for the trac ticket - that is quite interesting, I wasn't aware of that. But since that is a WordPress.org issue, it isn't something that is really actionable in this repository - and if we could keep the discussion focused on the issue outlined here that would be very much appreciated.

**Comment by seb86** [comment]
Thank you @timmyc Any chance you can still some how work on stopping WordPress from tracking user roles that are set to Customer at least if this is not enabled? Happy to open a new ticket if needed.

**Comment by timmyc** [comment]
> Any chance you can still some how work on stopping WordPress from tracking user roles that are set to Customer at least if this is not enabled?

I personally don't have much experience with that part of WordPress core, but perhaps ask in WPORG slack maybe?

**Comment by justlevine** [comment]
The language could be switched to opt-in.
> Opt In: Checking this box will tell us your store exists, and permit us to collect usage data. 

Even better: switch it to a toggle with state-specific description.
>Opt Out: Unchecking this box will prevent us from knowing your store exists, and we will not collect any usage data.

**Comment by DavidAnderson684** [comment]
> Unchecking this box will prevent us from knowing your store exists

I'll re-post the substance of the comment that I posted over on the other thread, so that it doesn't get lost. AFAIK, this statement is only true if we assume that the user will understand a particularly subtle meaning of "us".

By default, WP sends back the store URL, active plugins (i.e. WooCommerce) and number of users (an excellent proxy in most situations for number of customers, and daily growth thereof) to wordpress.org daily, to which (again, AFAIK, some) Automattic employees have access. I've read the opinion of Heather Burns (I can't find the URL now), probably the most respected voice on WordPress data protection issues, that ultimately the division between Automattic and wordpress.org in terms of legal control is only a technical, on-paper division. @webdevlaw, is that a fair statement your view? (Not sure if @-mentions work if someone's not contributed to a project, but I've tried).

Given that, how can this be worded so as to be accurate and so that the user is fully informed (which in some jurisdictions, e.g. EU under the GDPR, is more than just a matter of courtesy) ?

**Comment by webdevlaw** [comment]
Hi David,

I just received an email with your message, so @-mentions to non-contributors do work.

I don't recall exactly where the comment you're referencing was made, but I do recall it would have been in the context of the governance relationship between the Foundation and the dot org project. It should not be repurposed out of context into this issue.

In any case, Automattic had my time and labour on Jetpack and Woocommerce GDPR issues last year, and chose not to respect that time or my contributions. You will appreciate, then, that this is an issue I have no desire to be dragged into from any angle.

If there any issues relating to .org's handling of privacy issues which we can assist with, please do raise a ticket with the component tag _privacy_ and let the core-privacy team know in our Slack channel. We have an outstanding team doing outstanding work, and we are always looking for new contributors.

Thank you for your understanding.

**Comment by DavidAnderson684** [comment]
Hi Heather, apologies - I knew nothing of all that!

**Comment by justlevine** [comment]
>I'll re-post the substance of the comment that I posted over on the other thread, so that it doesn't get lost. AFAIK, this statement is only true if we assume that the user will understand a particularly subtle meaning of "us".

I was typing too quickly. Was focusing on the double negatives and sentence structure you mentioned. Editing the above to replace we/us with WC

**Comment by timmyc** [comment]
Fix pending in #23216
